### PR TITLE
Feature/fwf 3703 reusable build modal

### PR DESCRIPTION
**Pull request by Ajay-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3703
Issue Type: BUG/ FEATURE

# Changes
reusable build modal added
image
image

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-components/src/components/CustomComponents/BuildModal.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <>
+      {contents.map(({ id, heading, body, onClick }) => (
+        <div
+          className="col-md-6 build-contents"
+          key={id}
+          onClick={onClick}
+          role="button"
+          tabIndex={0}
+          onKeyDown={(event) => handleKeyDown(event, onClick)}
+        >
```
Comment (arun-s-aot) [inline forms-flow-components/src/components/CustomComponents/BuildModal.tsx:40]: can u pls add data test id here. in this case u can try a data test id with below format 

${heading}-option

Comment (Ajay-aot) [reply forms-flow-components/src/components/CustomComponents/BuildModal.tsx]: added

File: forms-flow-theme/scss/_modal.scss [hunk 2]
```diff
@@ -557,3 +557,33 @@
  gap: var(--spacer-150);
  align-self: stretch;
 }
+.build-modal{
+  width: 60.318% !important;
+  max-width: 60.318% !important;
```
Comment (arun-s-aot) [inline forms-flow-theme/scss/_modal.scss:562]: may be we have to update this on later stage. will merge for now @Ajay-aot

**Review by arun-s-aot — APPROVED** [review]

File: forms-flow-theme/scss/_modal.scss (hunk 2 above)
Comment (Ajay-aot) [reply forms-flow-theme/scss/_modal.scss:562]: okay

**Pull request merged**